etc/policy.template.toml

```diff
@@ -0,0 +1,338 @@
+# they expire; see 'signature-remain-time' to control regeneration time.
+#
+# An integer value is interpreted as seconds.
+signature-lifetime = 1209600 # TODO: "2w"
+
+# The amount of time remaining before expiry when signatures will be
+# regenerated.
+#
+# In order to prevent a zone's signatures from appearing invalid, they
+# have to be regenerated before they expire.  That hard limit is set by
+# 'signature-lifetime' above.  This setting controls how long before expiry
+# signatures will be regenerated; it must be less than the 'signature-lifetime'
+# setting.
+#
+# An integer value is interpreted as seconds.
+signature-remain-time = 604800 # TODO: "1w"
+
+# How denial-of-existence records are generated.
+[signer.denial]
+
+# The type of denial-of-existence records to generate.
+#
+# Supported options:
+# - 'nsec': Use NSEC records (RFC 4034).
+# - 'nsec3': Use NSEC3 records (RFC 5155).
+type = "nsec"
+
+# Whether to skip NSEC3 records for unsigned delegations.
+#
+# This enables the NSEC3 Opt-Out flag, and skips delegations to unsigned zones
+# when generating NSEC3 records.  This is a performance optimization for large
+# zones with many unsigned delegations, but it should not be enabled carelessly.
+#opt-out = false
+
+# How signed zones are reviewed.
+[signer.review]
+
+# Whether review is required.
+#
+# If this is 'true', a signed version of a zone will not be published until the
+# user explicitly approves it.  If it is 'false', signed zones will be published
+# immediately.  The review hook (below) will be run in any case.
+required = false
+
+# A hook for reviewing a signed zone.
+#
+# This command string will be executed in the user's shell when a new version of
+# a zone is signed.  At the moment, it will only be run if 'required' is true.
+#
+# It will receive the following information via environment variables:
+#
+# - 'CASCADE_ZONE': The name of the zone, formatted without a trailing dot.
+# - 'CASCADE_SERIAL': The serial number of the signed zone (decimal integer).
+# - 'CASCADE_UNSIGNED_SERIAL': The serial number of the unsigned zone from which
+#   the signed zone was generated (decimal integer).
+# - 'CASCADE_TOKEN': A UUID associated with this review, for approving or
+#   rejecting this version of the zone.
+# - 'CASCADE_SERVER': The TCP/UDP port where Cascade is serving the zone for
+#   review, formatted '<ip-addr>:<port>'.
+# - 'CASCADE_UNSIGNED_SERVER': The TCP/UDP port where Cascade is serving the
+#   unsigned version of the zone for review, formatted '<ip-addr>:<port>'.
+# - 'CASCADE_CONTROL': The address of Cascade's HTTP API server, for sending
+#   approvals and rejections.
+#
+# To approve the zone contents, send an HTTP POST request to
+# `http://$CASCADE_SERVER/rs2/approve/$CASCADE_TOKEN?zone=$CASCADE_ZONE&serial=$CASCADE_SERIAL`
+# (where `approve` can be replaced by `reject` to reject the zone).
+#
+# TODO: Use the exit code of the hook to infer approval / rejection?
```

Comment:
Maybe we should say that the cmd-hook should refer to a script that the user running Cascade has access to, should be executable and should POST to the review HTTP endpoint before it exits?